docs/source/contents/4_draft_mapping.rst

```diff
@@ -113,7 +113,7 @@ Justification
 ^^^^^^^^^^^^^
 
 These values are being kept because they may be helpful to users in finding specific materials. For instance, while :code:`@type="pid"`
-identifiers will no longer be the primary identifiers on UTK's next digital collections platform, they could be used to
+identifiers will no longer be the primary identifiers on UT's next digital collections platform, they could be used to
 identify cited resources that have broken links. Many of the identifiers associated with Special Collections allow users
 to see how the same resource might be referenced within finding aids. Have :code:`@type="opac"` identifiers helps staff at UTK
```

Comment:
UTK -> UT?

Reply:
Thanks for this. Yes, I had intended to change all for consistency and then didn't. For a bit I was reading a section where it was mostly UT so I went with that, though looking now it likely would've been easier to have kept with UTK and changed the other way. I should have addressed all instances of "UTK" in my most recent commit.